client/components/AvailabilityGrid/AvailabilityGrid.js

```diff
@@ -198,12 +198,11 @@ class AvailabilityGrid extends React.Component {
 
       participants.forEach((participant) => {
         if (participant.availability.indexOf(cellFormatted) > -1) {
-          availableOnDate.push(participant.name);
+          availableOnDate.push(participant.userId.name);
```

Comment:
at the db they are different documents, but I populate this info at the querys. so i can use as a single document at client side.